app/uk/gov/hmrc/ngrraldfrontend/connectors/NGRConnector.scala

```diff
@@ -51,7 +51,7 @@ class NGRConnector @Inject()(http: HttpClientV2,
     getPropertyLinkingUserAnswers(credId)
       .map {
         case Some(propertyLinkingUserAnswers) => Some(propertyLinkingUserAnswers.vmvProperty)
-        case None => throw new NotFoundException("failed to find propertyLinkingUserAnswers from backend mongo")
+        case None => None
```

Comment:
Just thinking if we deal with the exception here we don't have to do it anywhere else this is called. And maybe in the future we should show that page was talking about to not show kibana errors.